docs/fidesops/docs/guides/database_connectors.md

```diff
@@ -29,7 +29,7 @@ Other platforms will be added in future releases.
 
 ## Creating a ConnectionConfig object 
 
-The connection between Fidesops and your database is represented by a _ConnectionConfig_ object. To create a ConnectionConfig, you issue a request to the [Create a ConnectionConfig](/api#operations-Connections-get_connections_api_v1_connection_get) operation, passing a payload that contains the properties listed below. 
+The connection between Fidesops and your database is represented by a _ConnectionConfig_ object. To create a ConnectionConfig, you issue a request to the [Create a ConnectionConfig](/fidesops/api#operations-Connections-get_connections_api_v1_connection_get) operation, passing a payload that contains the properties listed below. 
```

Comment:
We can specifically link to the create/update endpoint - `/fidesops/api/#operations-Connections-put_connections_api_v1_connection_put`